api/src/main/java/org/pmiops/workbench/api/DiskAdminController.java

```diff
@@ -14,6 +14,7 @@
 @RestController
 public class DiskAdminController implements DiskAdminApiDelegate {
 
+  // TODO: switch to DiskAdminService?
```

Comment:
this controller calls 2 methods which I think would be appropriate for DiskAdminService as well.  Should I move them in this PR?